src/storage/mongodb/TagStorage.ts

```diff
@@ -39,6 +39,118 @@ export default class TagStorage {
     await Logging.traceEnd(tenantID, MODULE_NAME, 'saveTag', uniqueTimerID, tagMDB);
   }
 
+  public static async saveImportedTag(tenantID: string, importedTagToSave: ImportedTag): Promise<void> {
+    const uniqueTimerID = Logging.traceStart(tenantID, MODULE_NAME, 'saveImportedTag');
+    const tagMDB = {
+      _id: importedTagToSave.id,
+      description: importedTagToSave.description,
+      importedOn: new Date(),
```

Comment:
We should reuse the import date linked to the import user and not creating a new one each time.

Reply:
Done